docs/pages/reference/cli.mdx

```diff
@@ -387,6 +387,113 @@ $ tsh play --format=pty 1fe153d1-ce8b-4ef4-9908-6539457ba4ad.tar
 $ tsh play --format=json ~/play/0c0b81ed-91a9-4a2a-8d7c-7495891a6ca0.tar | jq '.event
 ```
 
+### tsh proxy db
+
+Start a local TLS proxy for database connections when using Teleport with TLS
+Routing enabled. Clients can connect to a Teleport-registered database through
+the local proxy.
+
+```code
+$ tsh proxy db [<flags>] <db>
+```
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--cert-file` | none | string | Path to the certificate file for configuring TLS on the local proxy.|
+| `--cluster` | none | string | The name of the Teleport cluster to connect to. |
+| `--key-file` | none | string | Path to the private key file for configuring TLS on the local proxy.|
+| `--db-name` | none | string | Optional database name to log in to. |
+| `--db-user` | none | string | Optional database user to log in as. |
+| `--port` | string | Source port used by the local proxy.|
+| `--tunnel` | none | Boolean | Open an authenticated tunnel using a database's client certificate so clients don't need to authenticate. |
+
+#### [Global Flags](#tsh-global-flags)
+
+These flags are available for all commands `--login, --proxy, --user, --ttl, --identity, --cert-format, --insecure, --auth, --skip-version-check, --debug, --jumphost, --format`.
+Run `tsh help <subcommand>` or see the [Global Flags Section](#tsh-global-flags)
+
+#### Examples
+```code
+$ tsh proxy db <db>
+
+# Proxy a connection to mysql db on local port 10700
+$ tsh proxy db --port 10700  mysql-db
+# Started DB proxy on 127.0.0.1:10700
+
+# Use following credentials to connect to the mysql-db proxy:
+#  ca_file=/Users/jeff/.tsh/keys/teleport.example.com/cas/teleport.example.com.pem
+#  cert_file=/Users/jeff/.tsh/keys/teleport.example.com/jeff-db/tele1c/mysql-db-x509.pem
+#  key_file=/Users/jeff/.tsh/keys/teleport.example.com/jeff
+
+# Proxy a connection to mysql db with no credentials required
+$ tsh proxy db --tunnel mysql-db
+Started authenticated tunnel for the MySQL database "mysql-db" in cluster "teleport.example.com" on 127.0.0.1:49415.
+
+Use the following command to connect to the database:
+  $ mysql --port 49415 --host localhost --protocol TCP
```

Comment:
Looks like there's a mix of docs-as-comments and regular un-commented copy(L434) in this code block. Would it make sense to elevate all copy and have separate code blocks?

Reply:
> 



> Changing my review status back. I incorrectly assumed you addressed all comments in my previous review with your last set of changes.

@alexfornuto should be all addressed now.  I turned off auto-merge